### PR TITLE
Implementation of strip_class_variables. More robust test cases.

### PR DESCRIPTION
**Pull request by haluzpav**
(Sorry about the previous PR I closed)

Tell me if it makes sense to provide option to strip class variables. I find it pretty useful in my application. This current implementation is able to handle also class inheritance.

I also made only one class for obj dump tests. They are maybe a bit less readable now, but more robust IMHO.

This PR also includes the previous one #5.

### CODE REVIEW

File: jsons/serializers.py [hunk 1]
```diff
@@ -158,28 +159,53 @@ def default_object_serializer(obj: object,
     private attributes (i.e. attributes that start with an underscore).
     :param strip_properties: if ``True`` the resulting dict will not contain
     values from @properties.
+    :param strip_class_variables: if ``True`` the resulting dict will not
+    contain values class variables.
```
Comment (ramonhagenaars) [inline jsons/serializers.py]: You forgot 'from'. :-)
```suggestion
    contain values from class variables.
```

File: jsons/serializers.py [hunk 2]
```diff
@@ -158,28 +159,53 @@ def default_object_serializer(obj: object,
...
+    props = []
+    other_cls_vars = []
+    for n, v in _get_complete_class_dict(cls).items():
+        props.append(n) if type(v) is property else other_cls_vars.append(n)
+    return props, other_cls_vars
+
+
+def _get_complete_class_dict(cls):
+    cls_dict = {}
+    for parent_class in reversed(cls.mro()):
```
Comment (ramonhagenaars) [inline jsons/serializers.py]: The use of `reversed` here is a nice one! It is subtle though, and yet quite important. Maybe we should spend a comment that explains why it is there? Also, `parent_class` does not fully cover as `cls` itself is contained too.

How about something like this:

```suggestion
    # Loop reversed so values of sub-classes override those of super-classes.
    for cls_or_elder in reversed(cls.mro()):
```

File: setup.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
     long_description = fh.read()
 
 setup(name='jsons',
-      version='0.5.4',
+      version='0.5.5',
```
Comment (ramonhagenaars) [inline setup.py]: Since you've added a nice feature as well, I think we should increase the version as follows:

```suggestion
      version='0.6.0',
```

**Comment by haluzpav** [comment]
Thanks. All done.

**Pull request merged**